src/ray/util/logging.h

```diff
@@ -85,6 +85,18 @@ enum { ERROR = 0 };
 #endif
 
 namespace ray {
+constexpr char kLogKeyAsctime[] = "asctime";
+constexpr char kLogKeyLevelname[] = "levelname";
+constexpr char kLogKeyMessage[] = "message";
+constexpr char kLogKeyFilename[] = "filename";
+constexpr char kLogKeyLineno[] = "lineno";
+constexpr char kLogKeyComponent[] = "component";
+constexpr char kLogKeyJobID[] = "job_id";
+constexpr char kLogKeyWorkerID[] = "worker_id";
+constexpr char kLogKeyNodeID[] = "node_id";
+constexpr char kLogKeyActorID[] = "actor_id";
+constexpr char kLogKeyTaskID[] = "task_id";
```

Comment:
what about:

constexpr std::string_view

And, are these keys pre-defined? Can/Should users use custom keys?

Reply:
Users can provide custom keys. These are just common pre-defined keys that we will likely index.